frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/BaseResetBranchToRemoteAction.java

```diff
@@ -157,11 +178,9 @@ public void run(ProgressIndicator indicator) {
           // If `changes` is null the whole root will be refreshed
```

Comment:
Do we need to explicitly trigger `GitMacheteRepository` refresh here, or will it always be triggered automatically anyway?

Reply:
Good question. We are subscribed to the vcs changes somewhere. Does it work in this case?

Reply:
Yeah, it's seems to be unneeded.